css/index.css

```diff
@@ -3,144 +3,364 @@
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.about-page .middle-content {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  border: none;
+}
+
+.img-container {
+  width: 50%;
+}
+
+.img-container p,
+.img-container h2,
+.img-container a,
+.img-container img,
+.img-container div {
+  padding: 0;
+  width: 370px;
+  margin: 0 auto;
+  margin-bottom: 20px;
+}
+
+.img-container div {
+  border-bottom: 1px dashed black;
+  margin-bottom: 50px;
+}
+
+.img-container a {
+  display: block;
+  color: black;
+  text-decoration: none;
+  padding: 10px 25px;
+  border: black solid 2px;
+  border-radius: 10px;
+  text-align: center;
+  width: 50%;
+  margin-left: 15px;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  width: 12.5%;
+  height: 100px;
+  background: black;
+  margin: 20px 2.5%;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
 .bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+  display: flex;
+  margin: 0 2% 20px;
+  justify-content: space-around;
 }
 
 .bottom-content .text-container {
-    padding-right: 4%;
+  padding-right: 4%;
 }
 
 .bottom-content .text-container:last-child {
-    padding-right: 0;
+  padding-right: 0;
 }
 
 footer {
-    width: 100%;
-    background: black;
+  width: 100%;
+  background: black;
 }
 
 footer nav {
-    width: 60%;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 2%;
+  font-size: 14px;
 }
 
 footer nav a {
-    color: white;
-    text-decoration: none;
-}
\ No newline at end of file
+  color: white;
+  text-decoration: none;
+}
+
+/*
+- [ ] box1: `teal`
+- [ ] box2: `gold`
+- [ ] box3: `cadetblue`
+- [ ] box4: `coral`
+- [ ] box5: `crimson`
+- [ ] box6: `forestgreen`
+- [ ] box7: `darkorchid`
+- [ ] box8: `hotpink`
+- [ ] box9: `indigo`
+- [ ] box10: `dodgerblue`
+*/
+
+.middle-content .boxes .box:nth-of-type(1) {
+  background: teal;
+}
+
+.middle-content .boxes .box:nth-of-type(2) {
+  background: gold;
+}
+
+.middle-content .boxes .box:nth-of-type(3) {
+  background: cadetblue;
+}
+
+.middle-content .boxes .box:nth-of-type(4) {
+  background: coral;
+}
+
+.middle-content .boxes .box:nth-of-type(5) {
+  background: crimson;
+}
+
+.middle-content .boxes .box:nth-of-type(6) {
+  background: forestgreen;
+}
+
+.middle-content .boxes .box:nth-of-type(7) {
+  background: darkorchid;
+}
+
+.middle-content .boxes .box:nth-of-type(8) {
+  background: hotpink;
+}
+
+.middle-content .boxes .box:nth-of-type(9) {
+  background: indigo;
+}
+
+.middle-content .boxes .box:nth-of-type(10) {
+  background: dodgerblue;
+}
```

Comment:
For the CSS the rule brick was looking for an organization of the css into sections. This is commonly done by making separate files or labeling separate sections within the HTML file.